src/xrpld/app/tx/detail/Transactor.cpp

```diff
@@ -972,6 +994,10 @@ Transactor::operator()()
             removeDeletedTrustLines(
                 view(), removedTrustLines, ctx_.app.journal("View"));
 
+        if (result == tecEXPIRED)
```

Comment:
Ditto here - IMO this should go into the above `result == tecEXPIRED` if statement a few lines above.

Reply:
Better not  to merge independent functionalities.